datadog_checks_base/datadog_checks/base/utils/__init__.py

```diff
@@ -8,7 +8,12 @@
     if datadog_agent.get_config('integration_tracing'):
         from ddtrace import patch
 
-        patch(requests=True)
+        # handle thread monitoring as an additional option
+        if datadog_agent.get_config('integration_tracing_futures'):
```

Comment:
Is that an existing configuration? Any reason not to patch futures by default?

Reply:
In cases like the SNMP check, monitoring threads can make for very large traces so it seemed like it would be helpful to leave that as an option if the area of performance inspection was elsewhere.